object-cache.php

```diff
@@ -1275,7 +1277,7 @@ public function build_client_parameters( $redis_server ) {
 		// 1s timeout, 100ms delay between reconnections.
 
 		// merging the defaults with the original $redis_server enables any custom parameters to get sent downstream to the redis client.
-		return array_replace_recursive( $redis_server, $defaults );
+		return array_replace_recursive( $defaults, $redis_server );
```

Comment:
Can you explain what difference swapping these variables makes?

Reply:
For future selves, RTFM: http://www.php.net/manual/en/function.array-replace-recursive.php